### PR TITLE
Move gifs to top in Desktop and CLI Pages courses

### PR DESCRIPTION
**Pull request by brianamarie**
## Overview
**TL;DR**
This pull request will move the gifs from the "show me how" sections up to the top, like we've done in the more recent [GraphQL](https://services.github.com/on-demand/graphql/) and [Electron](https://services.github.com/on-demand/electron/) courses. 

Additionally, to make maintenance easier in the future, this pull request removes some of the 'includes' for activities that are only used in one course. 

### Next Steps
- [x] Move gifs for GitHub 102 (Desktop)
- [x] Move gifs for GitHub 103 (CLI)
- [x] Move the `_includes/activities` so only activities used multiple times are included from another file

### Review
This isn't ready for review yet, but I will mention the training team when it's time. 👍

### CODE REVIEW

**Comment by brianamarie** [comment]
@github/services-training This is ready for review 🎉

**Comment by brianamarie** [comment]
Yay! Thanks @hollenberry!
